schemas/app-interface/app-interface-settings-1.yml

```diff
@@ -48,6 +48,12 @@ properties:
         type: string
       credentials:
         "$ref": "/common-1.json#/definitions/vaultSecret"
+  githubRepoInvites:
+    type: object
```

Comment:
Please add descriptions for the new fields.

Reply:
Fixed

Reply:
No, not that way 😄  A description field that documents the new field, e.g. https://github.com/app-sre/qontract-schemas/blob/main/schemas/app-sre/slo-document-1.yml#L42

Reply:
I see. Adjusted it now :)